testpmd/hooks/install.yml

```diff
@@ -1,12 +1,48 @@
 ---
+- name: Get all RuntimeClass
+  community.kubernetes.k8s_info:
+    api_version: node.k8s.io/v1
+    kind: RuntimeClass
+  register: rclass
+
+- name: Select High performance RuntimeClass
+  set_fact:
+    runtime_class: "{{ rclass | json_query(query) | first }}"
+  vars:
+    query: "resources[?handler=='high-performance'].metadata.name"
+  when: rclass|length > 0
 
 - name: Deploy the Example CNF applications
   include_role:
     name: "{{ deploy_dir }}/roles/example-cnf-app"
+  vars:
+    high_perf_runtime: "{{ runtime_class }}"
+
+- name: "Patch deployment to add annotations to testpmd pods"
+  community.kubernetes.k8s:
+    state: present
+    definition:
+      api_version: apps/v1
+      kind: Deployment
+      metadata:
+        name: "testpmd-app"
+        namespace: "{{ cnf_namespace }}"
+      spec:
+        template:
+          metadata:
+            annotations:
+              irq-load-balancing.crio.io: "disable"
+          spec:
+            runtimeClassName: "{{ runtime_class }}"
+  when:
+    - runtime_class is defined
+    - runtime_class|length
 
 - name: Apply configuration for CNF Cert Suite execution
   include_role:
     name: "{{ dci_config_dir }}/hooks/roles/apply-cnf-cert-config"
+  vars:
```

Comment:
why this variable is needed here?

Reply:
It is not used anymore, let me remove this